pkg/bwagreement/test/common.go

```diff
@@ -84,3 +90,20 @@ func GenerateRenterBandwidthAllocation(pba *pb.PayerBandwidthAllocation, uplinkK
 		Data:      data,
 	}, nil
 }
+
+// get uplink's public key
+func getUplinkPubKey(uplinkKey crypto.PrivateKey) ([]byte, error) {
```

Comment:
nit - rename to `getPublicKey` since this is a generic function. It can take in any `crypto.PrivateKey`, it does not need to be Uplink's